### PR TITLE
feat: add project settings and marker manager to global show list

### PR DESCRIPTION
**Pull request by elgiovanno**
Let me know what you think. As always thank you for this gamechanging script!

### CODE REVIEW

File: definitions/actions.lua [hunk 1]
```diff
@@ -278,6 +278,7 @@ return {
       PrevTransientInItem = {"SaveItemSelection", "SelectItemsUnderEditCursor", "MoveEditCursorToPrevTransientInSelectedItems", "RestoreItemSelection"},
       ProjectEnd = custom.move.projectEnd,
       ProjectStart = custom.move.projectStart,
+      ProjectSettings = 40021,
```
Comment (gwatcha) [inline definitions/actions.lua:281]: Hey thanks for doing this! I would rename ProjectSettings to 'ShowProjectSettings' though, nouns are used for selecting tracks / parts of the timeline. Take a look at the top of actions.lua to see some notes about naming conventions.

Once you change that it should be all good

Comment (elgiovanno) [reply definitions/actions.lua:281]: Yep! I'll fix that right up. Also, I wonder if ShowProjectSettings would be more useful in the <SPC> project folder? Or do you prefer it where it is now?

Comment (gwatcha) [reply definitions/actions.lua:281]: Yeah I'd put it in the project folder, probably under '<SPC>p,'!

Comment (elgiovanno) [reply definitions/actions.lua:281]: 'p' Is taken there by PreviousTab :/

Comment (elgiovanno) [reply definitions/actions.lua:281]: 'P'??

Comment (elgiovanno) [reply definitions/actions.lua:281]: Oh wait... Hmmm. There seem to be two I stances of PrevTab. One is 'N' the other is 'P'

**Comment by gwatcha** [comment]
Alright thanks! I'll merge it

**Pull request merged**